### PR TITLE
Correct Jenkinsfile

### PR DESCRIPTION
**Pull request by alezor**
Jenkinsfile should now lead to the correct path
New Hono certificates
TestData files support

### CODE REVIEW

File: testing/src/test/java/org/eclipse/kuksa/testing/client/TestApiClient.java [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+ * Copyright (c)  2019 Assystem GmbH [and others].
  *
  * This program and the accompanying materials are made
  * available under the terms of the Eclipse Public License 2.0
  * which is available at https://www.eclipse.org/legal/epl-2.0/
  *
  * SPDX-License-Identifier: EPL-2.0
  *
- * Contributors: Expleo Germany GmbH
+ * Contributors: Assystem GmbH
```
Comment (rohoet) [inline testing/src/test/java/org/eclipse/kuksa/testing/client/TestApiClient.java]: This change is wrong, I guess?

File: testing/src/test/resources/result/Hono-TestSuite.yaml [hunk 2]
```diff
@@ -1,15 +1,51 @@
 testCases:
   testGetTenantInfo:
+    testData:
+    - tenantId: ASSYSTEM_TENANT4
```
Comment (rohoet) [inline testing/src/test/resources/result/Hono-TestSuite.yaml]: Should this be changed ?

File: testing/src/test/resources/result/Hono-TestSuite.yaml [hunk 3]
```diff
@@ -1,15 +1,51 @@
...
+    - username: assystem2@ASSYSTEM_TENANT4
+      password: whySoSecret
+      body: '{"temperature":"5"}'
+    results:
+    - statusCode: '202'
+
+  testPostEventData:
+    testData:
+    - body: '{"weather":"snow"}'
+      username: assystem2@ASSYSTEM_TENANT4
```
Comment (rohoet) [inline testing/src/test/resources/result/Hono-TestSuite.yaml]: Change to Expleo?

File: testing/src/test/java/org/eclipse/kuksa/testing/client/TestApiClient.java (hunk 1 above)
Comment (alezor) [reply testing/src/test/java/org/eclipse/kuksa/testing/client/TestApiClient.java]: Whoops, yes something went wrong there. Thanks for pointing out

File: testing/src/test/resources/result/Hono-TestSuite.yaml (hunk 2 above)
Comment (alezor) [reply testing/src/test/resources/result/Hono-TestSuite.yaml]: The test data is thought to be dynamic and temporary for the biggest part of the tests, so it doesn't really matter for us. I will change it to some generic names in the future, however

File: testing/src/test/resources/result/Hono-TestSuite.yaml (hunk 3 above)
Comment (alezor) [reply testing/src/test/resources/result/Hono-TestSuite.yaml]: It's fine for now i guess

**Pull request merged**